src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/candidate/statusTag.tsx

```diff
@@ -39,10 +39,10 @@ function StatusTag({candidate}: Props) {
       const {details} = download;
       return (
         <StatusTagTooltip
-          label={t('No debug file was not found at this location.')}
+          label={t('No debug file was not found at this location:')}
```

Comment:
We should move `:` out of the translation.

Reply:
I can do that, but please why? I noticed that we usually use "?" and "."